projects/mobius/index.js

```diff
@@ -0,0 +1,144 @@
+/*==================================================
+  Modules
+  ==================================================*/
+const sdk = require("@defillama/sdk");
+const BigNumber = require("bignumber.js");
+const _ = require("underscore");
+const { getBlock } = require("../helper/getBlock"); //added module
+
+/*==================================================
+  Addresses
+  ==================================================*/
+const wBTC = "0x2260fac5e5542a773aa44fbcfedf7c193bc2c599";
+const wETH = "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2";
+const cUSD = "0x765DE816845861e75A25fCA122bb6898B8B1282a";
+const wBTCO = "0xBe50a3013A1c94768A1ABb78c3cB79AB28fc1aCE";
+const pUSDC = "0xcC82628f6A8dEFA1e2B0aD7ed448bef3647F7941";
+const cUSDC = "0x2A3684e9Dc20B857375EA04235F2F7edBe818FA7";
+
+const decimals = {
+  [wBTC]: 8,
+  [wBTCO]: 8,
+  [cUSDC]: 6,
+  [pUSDC]: 6,
+  ["celo-dollar"]: 0,
+};
+
+// {[string: name]: {
+//   address: string,
+//   peggedTo: string
+//   tokens: string[],
+// }}
+const pools = {
```

Comment:
any way we can get these automatically?

Reply:
The way the bridge works, they send the headers like decimals, name, and symbol not with every transaction but with a batch. They haven't been updated yet for all of them so we can't read the correct values from the contracts

Reply:
got it! we will be relying on the community to keep it updated through PRs then :)